docker/docker-compose-codebook_env.yaml

```diff
@@ -0,0 +1,15 @@
+# docker-compose up --build
+version: '2.4'
+services:
+  codebook_env:
+    build: # only necessary if rebuilding image
+      context: .
+      dockerfile: codebook.Dockerfile
+      args:
+        - JHUB_VER
+        - PY_VER
+        - DIST
+        - DEPLOY_KEY
+        - REPO_OWNER
+        - REPO_NAME
+    image: registry.vathes.com/${REPO_OWNER}/codebook-${REPO_NAME}:jhub${JHUB_VER}-py${PY_VER}-${DIST}-v${WORKFLOW_VERSION}
```

Comment:
After discussion with @yambottle @guzman-raphael, we will publish to open-source DataJoint DockerHub.